dhctl/pkg/kubernetes/actions/deckhouse/delete.go

```diff
@@ -38,6 +38,18 @@ const (
 	deckhouseDeploymentName      = "deckhouse"
 )
 
+type StorageCRsConfig struct {
+	Resources  []string
+	ApiGroup   string
+	ApiVersion string
+}
+
+var d8storageConfig = StorageCRsConfig{
+	Resources:  []string{"localstorageclasses", "replicatedstorageclasses", "nfsstorageclasses", "cephstorageclasses"},
+	ApiGroup:   "storage.deckhouse.io",
+	ApiVersion: "v1alpha1",
+}
```

Comment:
You would probably want this to be stored as a slice of [GroupVersionResource](https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime/schema#GroupVersionResource) instead at some point in the future. For when new api versions will be pushed.